content/docs/command-reference/params/index.md

```diff
@@ -143,6 +143,14 @@ $ dvc run -n train -d logs/ -o users.csv \
           python train.py
 ```
 
+Alternatively, JSON or TOML can be used to specify the parameters file:
+
+```dvc
+$ dvc run -n train -d logs/ -o users.csv \
+          -p params.toml:threshold,classes_num \
+          python train.py
+```
+
```

Comment:
Thanks but probably not needed, it's repetitive and we already have a note reminding the user that JSON and TOML are also supported. So:
```suggestion
```